websites/S/Super Animes/iframe.ts

```diff
@@ -0,0 +1,81 @@
+var iframe = new iFrame();
+
+iframe.on("UpdateData", async () => {
+    if (document.querySelector("#play0") != null) {
+        const video: HTMLVideoElement = document.querySelector("#play0");
+        if (video != null) {
+            const played = video.duration != 0;
+            iframe.send({
+                currentTime: video.currentTime,
+                duration: video.duration,
+                played,
+                paused: video.paused
+            });
+        }
+    }
+
+    if (document.querySelector("#play1_html5_api") != null) {
+        const video: HTMLVideoElement = document.querySelector("#play1_html5_api");
+        if (video != null) {
+            const played = video.duration != 0;
+            iframe.send ({
+                currentTime: video.currentTime,
+                duration: video.duration,
+                played,
+                paused: video.paused
+            });
+        }
+    }
+
+    if (document.querySelector("#video") != null) {

```

Comment:
You don't have to do it like this all the time. `querySelector` won't throw an error if it couldn't find the element. It'd return `null` so you can use the or operator (`||`) without any problem. This will save you from the unnecessary code and will beautify your code. Example:

```ts
const video: HTMLVideoElement = document.querySelector("#play0") || document.querySelector("#video")

if (video && !isNaN(video.currentTime) iframe.send({});
```

Reply:
or just 
`const video: HTMLVideoElement = document.querySelector("video")`